plugins/woocommerce/i18n/states.php

```diff
@@ -1168,6 +1168,25 @@
 		'SAG' => __( 'Sagarmatha', 'woocommerce' ),
 		'SET' => __( 'Seti', 'woocommerce' ),
 	),
+	'NI' => array( // Nicaragua states
+		'NI-AN' => __( 'Costa Caribe Norte', 'woocommerce' ),
+		'NI-AS' => __( 'Costa Caribe Sur', 'woocommerce' ),
```

Comment:
Let's change these to [Atlántico](https://github.com/unicode-org/cldr/blob/release-40/common/subdivisions/en.xml#L3965-L3966) per the CLDR.

Reply:
@jeffstieler done!